internalstaging.theanvil.io/manifest.json

```diff
@@ -7,27 +7,27 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2020.09",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2020.10",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
-    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:4.23.5",
```

Comment:
It looks like fence 4.23.5 is NOT in 2020.10 -- we may need to pin it, this ver of fence is needed for RAS stuff right? 
https://github.com/uc-cdis/fence/compare/2020.10...master
https://github.com/uc-cdis/fence/compare/4.23.5...master

Reply:
Yes, the automation doesn't make any distinction when we have semantic versions. 🤕 
good catch!